packages/clerk-js/src/ui/components/PricingTable/PricingTableDefault.tsx

```diff
@@ -175,16 +184,53 @@ function Card(props: CardProps) {
             />
           </Box>
         ) : null}
+
+        {showStatusRow && (
+          <Box
+            sx={t => ({
+              padding: t.space.$1,
+              borderTopWidth: t.borderWidths.$normal,
+              borderTopStyle: t.borderStyles.$solid,
+              borderTopColor: t.colors.$neutralAlpha100,
+            })}
+          >
+            {subscription?.status === 'active' || (isImplicitlyActiveOrUpcoming && subscriptions.length === 0) ? (
```

Comment:
This also feels weird that this is not gated behind `isSignedIn`, not sure why a not signed in user need to see the current plan

Reply:
I've added a check to only showing the status row if the user is signed in